dark/assets/css/styles.css

```diff
@@ -1,6 +1,7 @@
 /* Edo Font */

```

Comment:
Some early changes have been commited. Remove them @AlgyJr . Otherwise its perfect.

Reply:
Done

Reply:
Sorry, I can't understand what you want. Can you detail more?